taskvine/src/examples/vine_example.py

```diff
@@ -31,13 +31,13 @@
     gzip_path = "/usr/bin/gzip"
     if not os.path.exists(gzip_path):
       print("gzip was not found. Please modify the gzip_path variable accordingly. To determine the location of gzip, from the terminal type: which gzip (usual locations are /bin/gzip and /usr/bin/gzip)")
-      sys.exit(1);
+      sys.exit(1)
 
   # We create the tasks queue using the default port. If this port is already
   # been used by another program, you can try setting port = 0 to use an
   # available port.
   try:
-      q = Manager(port = VINE_DEFAULT_PORT)
+      q = vine.Manager(port = vine.VINE_DEFAULT_PORT)
```

Comment:
In all the examples, let's use:  `q = vine.Manager()`, as the default port is used when no arguments are given, and all caps constants look unpythonic.